diff --git a/certs/CMakeLists.txt b/certs/CMakeLists.txt
index 73e730f8..54106a3b 100644
--- a/certs/CMakeLists.txt
+++ b/certs/CMakeLists.txt
@@ -4,10 +4,10 @@
 
 # Create the certs directory. This is where the root CA cert will be installed.
 # Then copy the root CA cert to the certs directory.
-install(DIRECTORY DESTINATION ${CVD_CERTS_DIRECTORY} COMPONENT programs)
+install(DIRECTORY DESTINATION ${CMAKE_INSTALL_PREFIX}/${CVD_CERTS_DIRECTORY} COMPONENT programs)
 install(
     FILES
         ${CMAKE_CURRENT_SOURCE_DIR}/clamav.crt
     DESTINATION
-        ${CVD_CERTS_DIRECTORY}
+        ${CMAKE_INSTALL_PREFIX}/${CVD_CERTS_DIRECTORY}
     COMPONENT programs)
